src/main/java/seedu/address/logic/commands/FocusCommand.java

```diff
@@ -15,7 +15,10 @@
 public class FocusCommand extends Command {
 
     public static final String MESSAGE_FOCUS_CANDIDATE = "Details of candidate shown";
-    public static final String MESSAGE_USAGE = "focus [INDEX]";
+    public static final String MESSAGE_USAGE = "focus: Bring up a Candidate's information into the middle panel.\n"
```

Comment:
Only fix! But let me change it on my side.
```suggestion
    public static final String MESSAGE_USAGE = "focus: Brings up a Candidate's information into the middle panel.\n"
```